### ISSUE TITLE
Ways forward for this rubric

### ISSUE THREAD

**Issue by vongohren** [w3c/did-rubric#14]
Hi! I just wonder what the ways forward for this rubric is?
There is a lot of new DIDs and the openness of them, based on the criterias are worth maybe pointing out.
How does this work progress forward?

It has gotten shed new light on it, as DIF has launched a new FAQ with a question "How to choose a did method" where this spec is referenced.

Then I wonder how can this work progress?

**Comment by dhh1128** [comment]
Hi, @vongohren . I apologize for not responding to this issue earlier; I guess I wasn't getting the notifications for this repo that I expected.

The rubric is going to be proposed formally in CCG/DID WG circles as a "First Public Working Draft" document in the next few weeks. There is a large PR coming in that will incorporate a bunch of additional criteria that were discovered during the latest DHS SVIP collaborations. I didn't write this material, but I'm happy for it because I think it broadens the rubric in a nice way. If we can put that PR into the FPWG version of the rubric, great; otherwise, we'll try to associate it with a quick follow-on update.

Bottom line is that the timeline for this is getting late, so we should be maturing rapidly now. We are hoping to get a deeper engagement for feedback when we announce the status at CCG/DID WG meetings in the next few days.

**Comment by vongohren** [comment]
@dhh1128 okok! Thanks for, I will see if I catch the meetings and get the update :D
